superbit_lensing/galsim/mock_superBIT_data.py

```diff
@@ -448,6 +449,8 @@ def __init__(self, config_file, logprint, args=None):
         # Check that certain params are set either on command line or in config
         utils.check_req_params(self, self.__req_params, self.__req_defaults)
 
+        self._set_seeds()
```

Comment:
As mentioned above, it doesn't seem like this function runs correctly without a `master_seed` being set elsewhere

Reply:
It ran correctly in one of the meta config runs with the `master_seed` set explicitly. I will check again with a typical pipe test where each are set individually.

Reply:
Also, can you say more? Didn't run correctly in what way?

Reply:
Nevermind, bugs fixed. Full pipe test running now.